docs/libs/creating-lib.md

```diff
@@ -71,8 +71,8 @@ module.exports = function (config) {
     },
    reporters: ['progress', 'kjhtml', 'dots', 'junit'],
     junitReporter: {
-      outputFile: 'unit-test-results.xml',
-      outputDir: require('path').join(__dirname, '../../unit-tests/storefront'),      
+      outputFile: 'unit-test-storefront.xml',
+      outputDir: require('path').join(__dirname, '../../unit-tests-reports/storefront'),      
```

Comment:
from the direction that I'm now seeing, you you'll create a 1-1 relationship, where you create a directory and put that specific junit xml output to it.

Couldn't we have kept it in one folders with all these different outputFile instead?

for example,

junitReporter: {
 outputFile: 'unit-test-TODO:INSERTNAME.xml'
 outputDir: require(...).join(..., '../../unit-tests-reports'))
}

Basically, 1 folder, which contains all those output files